src/ui/components/tags/TagEditor.vue

```diff
@@ -0,0 +1,140 @@
+/*****************************************************************************
```

Comment:
Both the `TagEditor` and `TagSelection` are generic enough to be reused for plots, annotation inspectors, etc.